job-server/test/spark.jobserver/JobManagerSpec.scala

```diff
@@ -195,5 +196,20 @@ abstract class JobManagerSpec extends JobSpecBase(JobManagerSpec.getNewSystem) {
         case JobResult(_, result: Int) => result should equal (1 + 2 + 3)
       }
     }
+
+    it("should be able to cancel running job") {
```

Comment:
yes, I'm saying there's no test for that scenario, the user should receive a message that no such job exists right?